### PR TITLE
[seekbar-tapping] Fix always effect although be disabled in settings.

### PR DESCRIPTION
**Pull request by d4rkk3y**

### CODE REVIEW

**Comment by Canny1913** [comment]
Does this fix break the patch for old versions?

**Comment by d4rkk3y** [comment]
> Does this fix break the patch for old versions?

which version should I check? I checked the 17.32.35

**Comment by Canny1913** [comment]
Check 17.29.34.

**Comment by d4rkk3y** [comment]
Tested in 17.19.36,  17.29.34, 17.32.35. It work.

File: src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/patch/EnableSeekbarTappingPatch.kt [hunk 1]
```diff
@@ -85,6 +85,7 @@ class EnableSeekbarTappingPatch : BytecodePatch(
         val register = (instruction as Instruction35c).registerC
 
         // the instructions are written in reverse order.
+        val elseLabel = implementation.newLabelForIndex(result.patternScanResult!!.endIndex + 1)
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/patch/EnableSeekbarTappingPatch.kt:88]: Please fix the comment, it should be below the instruction

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Pull request merged**